### PR TITLE
Min spacing on assertion

### PR DESCRIPTION
**Pull request by hkalodner**

### CODE REVIEW

**Review by hkalodner — COMMENTED** [review]
Changes on top of mine look good to me

**Review by joshuacolvin0 — APPROVED** [review]
LGTM, just one error message comment

File: packages/arb-avm-cpp/cmachine/arbcore.go [hunk 1]
```diff
@@ -352,6 +352,15 @@ func (ac *ArbCore) GetLastMachine() (machine.Machine, error) {
 	return ret, nil
 }
 
+func (ac *ArbCore) GetLastMachineTotalGas() (*big.Int, error) {
+	result := C.arbCoreGetLastMachineTotalGas(ac.c)
+	if result.found == 0 {
+		return nil, errors.New("failed to load send count")
```
[the remark this answers is not in the document]
Comment (PlasmaPower) [reply packages/arb-avm-cpp/cmachine/arbcore.go]: Thanks, fixed

**Review by joshuacolvin0 — APPROVED** [review]
LGTM

**Pull request merged**